frontend/src/dashboard.js

```diff
@@ -246,6 +271,17 @@ export class Dashboard extends React.Component {
                   <PlusCircleIcon />
                 </Button>
               </FlexItem>
+              <FlexItem>
```

Comment:
Add ` spacer={{ default: 'spacerNone' }}` to the PREVIOUS `FlexItem` to get rid of the huge space between these two.